Makefile

```diff
@@ -85,20 +108,37 @@ linter:
 	go test $(PKGNAME)/core/container -run=BuildImage_Obcca
 	@touch $@
 
-.baseimage-dummy:
+.baseimage-dummy: $(BASEIMAGE_DEPS)
 	@echo "Building docker base-image"
-	@./scripts/provision/docker.sh 0.0.9
+	@./scripts/provision/docker.sh $(BASEIMAGE_RELEASE)
 	@touch $@
 
+# Special override for protoc-gen-go since we want to use the version vendored with the project
+gotool.protoc-gen-go:
+	mkdir -p $(GOPATH)/src/github.com/golang/protobuf/
+	cp -r $(GOPATH)/src/github.com/hyperledger/fabric/vendor/github.com/golang/protobuf/ $(GOPATH)/src/github.com/golang/
+	go install github.com/golang/protobuf/protoc-gen-go
+	rm -rf $(GOPATH)/src/github.com/golang/protobuf
+
+# Default rule for gotools uses the name->path map for a generic 'go get' style build
+gotool.%:
+	$(eval TOOL = ${subst gotool.,,${@}})
+	go get ${go.fqp.${TOOL}}
+
+$(GOPATH)/bin/%:
```

Comment:
Nevermind, my experiment failed to produce a better approach.  Good to go as is.